services/helper/bigBlueButton.js

```diff
@@ -0,0 +1,82 @@
+const bigBlueButtonUrl = process.env.BIG_BLUE_BUTTON_URL + process.env.BIB_BLUE_BUTTON_BASE_URL;
+const crypto = require("crypto");
+const request = require("../../generics/requests");
+const endpoints = require('../../constants/endpoints');
+
+module.exports = class SessionsHelper {
+    static async createMeeting(meetingId,meetingName,attendeePW,moderatorPW) {
+        try {
+
+            // let recordingCallBackUrl = encodeURI(process.env.RECORDING_READY_CALLBACK_URL);
+            // "&meta_bbb-recording-ready-url=" + recordingCallBackUrl;
+            let endMeetingCallBackUrl = process.env.MEETING_END_CALLBACK_EVENTS + "%2F" + meetingId;
+
+            meetingName = encodeURI(meetingName);
+            let query = "name=" + meetingName + "&meetingID=" + meetingId + "&record=true" + "&autoStartRecording=true" + "&meta_endCallbackUrl=" + endMeetingCallBackUrl + "&attendeePW=" + attendeePW + "&moderatorPW=" + moderatorPW;
+            let checkSumGeneration = "create" + query + process.env.BIG_BLUE_BUTTON_SECRET_KEY;
+            var shasum = crypto.createHash('sha1');
+            let sha = shasum.update(checkSumGeneration);
+            const checksum = sha.digest('hex');
+
+            const createUrl = bigBlueButtonUrl + endpoints.CREATE_MEETING + "?" + query + "&checksum=" + checksum;
+            let response = await request.get(createUrl);
+            return response.success;
+
+        } catch (error) {
+            throw error;
+        }
+    }
+
+    static async joinMeetingAsModerator(meetingId,mentorName,moderatorPW) {
+        try {
+
+            mentorName = encodeURI(mentorName);
+            let query = "meetingID=" + meetingId + "&password=" + moderatorPW + "&fullName=" + mentorName;
+            let checkSumGeneration = "join" + query + process.env.BIG_BLUE_BUTTON_SECRET_KEY;
+            var shasum = crypto.createHash('sha1');
+            shasum.update(checkSumGeneration);
+            const checksum = shasum.digest('hex');
```

Comment:
@aman-tunerlabs for checksum logic creation can be written in common function.

Reply:
@rakeshSgr what i think is this can be in bigBlueButton only but as a separate function as checksum logic is used in bigbluebutton only not across the module.